package.json

```diff
@@ -40,29 +42,20 @@
     "d3": "^4.9.1",
     "d3-scale-chromatic": "^1.1.1",
     "decimal.js": "^7.2.3",
-    "es6-promise": "4.0.5",
     "lodash": "4.17.4",
     "moment": "2.18.1",
     "moment-duration-format": "^1.3.0",
     "normalizr": "^3.2.2",
     "prop-types": "^15.6.0",
     "react": "15.6.1",
     "react-autosize-textarea": "^0.4.7",
-    "react-autosuggest": "^9.0.0",
-    "react-custom-scroll": "2.0.1",
     "react-datetime": "^2.10.1",
-    "react-dnd": "2.5.1",
-    "react-dnd-html5-backend": "2.5.1",
     "react-dom": "15.6.1",
-    "react-draggable": "3.0.3",
-    "react-intl": "2.4.0",
     "react-modal": "^2.4.1",
-    "react-portal": "^3.0.0",
     "react-redux": "^4.4.6",
     "react-router": "3.0.2",
     "react-router-redux": "4.0.7",
     "react-select": "^1.0.0-rc.10",
-    "react-textarea-autosize": "^4.0.5",
```

Comment:
I think this is used for the description of a market

Reply:
@andre-meyer there is `react-autosize-textarea` which is actually used